src/core/class_db.cpp

```diff
@@ -352,8 +352,8 @@ void ClassDB::initialize(GDExtensionInitializationLevel p_level) {
 }
 
 void ClassDB::deinitialize(GDExtensionInitializationLevel p_level) {
-	for (std::vector<StringName>::reverse_iterator i = class_register_order.rbegin(); i != class_register_order.rend(); ++i) {
-		const StringName &name = *i;
+	std::set<StringName> to_erase;
+	for (const auto& name : class_register_order) {
```

Comment:
The use of the keyword `auto` is [forbidden](https://docs.godotengine.org/en/latest/contributing/development/cpp_usage_guidelines.html#auto-keyword)

Reply:
Thanks, I'll fix it. Just note that I assumed it was fine here because of this https://github.com/godotengine/godot-cpp/blob/cd61a9bbe881b902750111eac8b02c04778f366b/src/core/class_db.cpp#L365
I'll fix it as well while I'm at it. I didnt look at the rest of the code.